.github/workflows/pythonpackage.yml

```diff
@@ -27,36 +29,39 @@ jobs:
 
     - name: Get files changed
       id: diff-files
+      uses: trilom/file-changes-action@v1.2.4
```

Comment:
good that someone else has already solved that... seems like there is already a lot of infrastructure available

Reply:
Yeah, the problem is sorting through the many out there to find the right one.  And with that I still had to filter the files in a complicated way.  Might be easier to clone it and modify it to our needs.

I've also [found one](https://github.com/tayfun/flake8-your-pr) now for flake8 that looks really good, but probably needs a couple of tweaks for us.  I'll be cloning it and experimenting with that...